Automattic/wp-calypso
# Atomic Hosting: Rename section to Hosting Configuration

#### Changes proposed in this Pull Request

* Renames the section to "Hosting Configuration".
* Changes the route to `/hosting-config/:site`.
* Applies some style adjustments to the layout so it matches the latest mockups. 

#### Screenshots

Atomic site with SFTP user created

| Viewport | Before | After |
|---|---|---
| Desktop | <img width="300" src="https://user-images.githubusercontent.com/1233880/69255923-ec072280-0bb8-11ea-850b-b2c3d6aac53e.png"> | <img width="300" src="https://user-images.githubusercontent.com/1233880/69330165-b9116d00-0c52-11ea-9af1-6c0a1d27252b.png"> |
| Tablet | <img width="200" src="https://user-images.githubusercontent.com/1233880/69257571-94b68180-0bbb-11ea-90b1-d23184892c70.png"> | <img width="200" src="https://user-images.githubusercontent.com/1233880/69330184-c3336b80-0c52-11ea-9994-99a68a8a388d.png"> |
| Mobile | <img width="100" src="https://user-images.githubusercontent.com/1233880/69257499-705aa500-0bbb-11ea-85e2-e0c4df1d2174.png"> |<img width="100" src="https://user-images.githubusercontent.com/1233880/69330214-ce869700-0c52-11ea-80d2-2d3fbe01107e.png"> |

Atomic site without SFTP user created

| Viewport | Before | After |
|---|---|---
| Desktop | <img width="300" src="https://user-images.githubusercontent.com/1233880/69256661-1c02f580-0bba-11ea-9587-2fd09c451222.png"> | <img width="300" alt="Screen Shot 2019-11-21 at 11 37 36" src="https://user-images.githubusercontent.com/1233880/69330581-7a2fe700-0c53-11ea-8f95-08ff7fd11c58.png"> |
| Tablet | <img width="200" src="https://user-images.githubusercontent.com/1233880/69257366-3ee1d980-0bbb-11ea-9389-61101b9b693d.png"> | <img width="200" alt="Screen Shot 2019-11-21 at 11 37 55" src="https://user-images.githubusercontent.com/1233880/69330607-861ba900-0c53-11ea-8ab9-937ffd1dbc66.png"> |
| Mobile | <img width="100" src="https://user-images.githubusercontent.com/1233880/69257304-296caf80-0bbb-11ea-80e3-8195e88c5075.png"> | <img width="100" alt="Screen Shot 2019-11-21 at 11 38 18" src="https://user-images.githubusercontent.com/1233880/69330648-9df32d00-0c53-11ea-8b2f-a4f1d9e22e51.png"> |

Business site no transferred to Atomic

| Viewport | Before | After |
|---|---|---
| Desktop | <img width="300" src="https://user-images.githubusercontent.com/1233880/69257093-c2e79180-0bba-11ea-8015-ccba47fdc328.png"> | <img width="300" alt="Screen Shot 2019-11-21 at 11 42 00" src="https://user-images.githubusercontent.com/1233880/69330854-0a6e2c00-0c54-11ea-92d1-6643f7040190.png"> |
| Tablet |<img width="200" src="https://user-images.githubusercontent.com/1233880/69257162-e579aa80-0bba-11ea-9ca3-a27037aff700.png"> | <img width="200" alt="Screen Shot 2019-11-21 at 11 42 21" src="https://user-images.githubusercontent.com/1233880/69330874-16f28480-0c54-11ea-9c77-00128f94c29e.png"> |
| Mobile | <img width="100" src="https://user-images.githubusercontent.com/1233880/69257211-fa563e00-0bba-11ea-86e1-f668767d29fd.png"> | <img width="100" alt="Screen Shot 2019-11-21 at 11 42 38" src="https://user-images.githubusercontent.com/1233880/69330904-240f7380-0c54-11ea-800c-cc71301cd07b.png"> |

#### Testing instructions

* Make sure the hosting section is displayed according to the screenshots above.
* Make sure the hosting section is named as "Hosting Configuration" (sidebar, browser tab name) and it is accessed through the `/hosting-config/:site` route.

Fixes #37585
Fixes #37584
